src/components/formikValidationSchema.ts

```diff
@@ -24,7 +24,7 @@ import { bytesToSensibleFormat } from '../util/fileSizeUtil';
 const appendError = (error: string, newError: string): string =>
   error ? `${error} \n ${newError}` : newError;
 
-interface RuleObject<FormikValuesType> {
+interface RuleObject<FormikValuesType, T> {
```

Comment:
Gjerne kall det andre generiske parameteret for noe mer beskrivende. Tror også det er lurt å legge inn en default-verdi, i og med at det ikke alltid er nødvendig. Kan sikkert være et tomt objekt eller `any`.